gen-ai-rag-vertex-ai-vector-search/README.md

```diff
@@ -0,0 +1,27 @@
+# Infrastructure for a RAG-capable generative AI application using Vertex AI and Vector Search
+
+This sample implements the [Infrastructure for a RAG-capable generative AI
+application using Vertex AI and Vector
+Search](https://cloud.google.com/architecture/gen-ai-rag-vertex-ai-vector-search)
+reference architecture from the [Cloud Architecture
+Center](https://cloud.google.com/architecture/).
+
+To try this sample, follow the [setup instructions](/README.md#setup).
+
```

Comment:
I suggest mentioning here the placeholders that users need to update before they can use the sample to try out the RAG flow: i.e., they should update the Cloud Run function and service references in ingestion.tf & serving.tf with their apps.